client/verta/verta/tracking/entities/_experimentrun.py

```diff
@@ -189,7 +198,11 @@ def _log_artifact(self, key, artifact, artifact_type, extension=None, method=Non
         extension : str, optional
             Filename extension associated with the artifact.
         method : str, optional
-            Serialization method used to produce the bytestream, if `artifact` was already serialized by verta.
+            Serialization method used to produce the bytestream, if `artifact`
+            was already serialized by Verta.
+        framework : str, optional
```

Comment:
Maybe this framework string should be an enum if the type is actually a small discrete set of values.

Reply:
As a follow-up, I am looking to implement it as an enum (or a class with constant attributes) as a part of https://github.com/VertaAI/docs/pull/68. I wanted to avoid packing it into this PR which seemed large enough as it is!